### PR TITLE
setup testing with react-testing-library and jest

### PR DESCRIPTION
**Pull request by suhaasya**
- installed necessary dependencies
- created jest.config.js and jest.setup.js files and added configuration from next js docs 
- created __tests__ folder added index.test.tsx file to check if setup is working

### CODE REVIEW

**Comment by suhaasya** [comment]
@bhtibrewal can you please review the pr?

File: package.json [hunk 1]
```diff
@@ -36,7 +42,9 @@
         "eslint-plugin-react": "^7.32.2",
         "eslint-plugin-react-hooks": "^4.6.0",
         "husky": "^8.0.3",
+        "jest": "^29.6.2",
+        "jest-environment-jsdom": "^29.6.2",
         "prettier": "^2.8.8",
-        "eslint": "8.41.0"
```
Comment (bhtibrewal) [inline package.json:40]: why has eslint been removed ?

File: jest.setup.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// Optional: configure or set up a testing framework before each test.
```
[the remark this answers is not in the document]
Comment (bhtibrewal) [reply jest.setup.js:1]: otherwise looks okay

[the remark this answers is not in the document]
Comment (suhaasya) [reply jest.setup.js]: removed that line.

[the remark this answers is not in the document]
Comment (bhtibrewal) [reply jest.setup.js]: I was talking about all the comments not just that particular line

[the remark this answers is not in the document]
Comment (suhaasya) [reply jest.setup.js]: sorry i misunderstood. removed them.

**Review by yesyash — APPROVED** [review]

**Review by bhtibrewal — APPROVED** [review]

**Pull request merged**